keras_core/distribute/distribution.py

```diff
@@ -0,0 +1,144 @@
+"""Unified high level distribution APIs across backends.
+
+!!!DO NOT USE!!! Currently under development and APIs are not final.
+
+Currently only the JAX backend has been implemented, and the Tensorflow backend
+will be implemented in future (via tf.dtensor API).
+"""
+
+import contextlib
+from typing import Any
+
+from keras_core import KerasVariable
+from keras_core.backend.common import global_state
+
+GLOBAL_ATTRIBUTE_NAME = "distribution"
+
+
+def list_devices(device_type: str = None) -> list[str]:
```

Comment:
We don't use type annotations elsewhere in the codebase -- just mention the argument types in their description in the docstring

Reply:
Sure. Somehow I feel the type annotation is more expressive than the docstring, but I will keep it consistent with the existing code.